qa/automation/CareerPortal.Tests.Scenarios/Savostin/CV_sending.feature

```diff
@@ -0,0 +1,179 @@
+Feature: 1.0.0 Sending CV
+Background: 
+	Given "Career Quantori" page is opened
```

Comment:
Same here: I would recommend to add URL (https://career.quantori.com/) instead of just the name of the website in order to avoid any misunderstanding in the future. :)

Reply:
Done)